image_explorer/public/js/image_explorer_v1.js

```diff
@@ -0,0 +1,124 @@
+function ImageExplorerBlock(runtime, element) {
```

Comment:
Yeah, I didn't try to optimise that too much since more differences will be incoming between them, this work is still in early stages. Image Explorer v1, on the other hand, won't be getting new features, only bug fixes probably, so its code is mostly frozen. 

The schema version can be injected, and different code paths can be taken on that basis since I'm not the original author of the code, I don't know if that will help though, because I don't know where this code will end up. And the desire to not break existing code is strong.

Reply:
Sounds good to me!